src/docs/asciidoc/management.adoc

```diff
@@ -794,6 +794,14 @@ Besides the Management Center's Hot Restart https://docs.hazelcast.org/docs/mana
 |===
 |**IMDG Open Source commands**
 
+* _Checking if a node is ready to use:_
++
+When a node is joined/formed a cluster and user can safely use it, the HTTP call below returns status code 200, otherwise it returns status code 503 to indicate node is not available yet.
++
+[source,plain]
+----
+curl --data "${GROUPNAME}&${PASSWORD}" http://127.0.0.1:${PORT}/hazelcast/health/ready
```

Comment:
@ahmetmircik I think we need to remove `--data "${GROUPNAME}&${PASSWORD}"` from documentation, since, as I see it is a `get` endpoint and curl `--data` sends the specified data in a `POST` request to the HTTP server,

Reply:
Thanks, should be fixed with https://github.com/hazelcast/hazelcast-reference-manual/pull/666